### DISCUSSION TITLE
关于美国国家民主基金会的一些事实清单

### DISCUSSION THREAD

**Discussion by lxn7022** [The-Run-Philosophy-Organization/run#421] [General]
[https://www.fmprc.gov.cn/web/wjb_673085/zzjg_673183/xws_674681/xgxw_674683/202205/t20220507_10683088.shtml](url)

![image](https://user-images.githubusercontent.com/66630801/167531740-1334b468-4a14-412f-95d4-72f67ed67cbd.png)
![image](https://user-images.githubusercontent.com/66630801/167531769-5c6f763d-605f-40bd-be38-6e87d752ee2a.png)
![image](https://user-images.githubusercontent.com/66630801/167531793-b8c3f535-36ac-4b11-ae31-d99a37ea8798.png)
![image](https://user-images.githubusercontent.com/66630801/167531822-cde725c8-ae4e-4265-9061-4a709e659923.png)
![image](https://user-images.githubusercontent.com/66630801/167531850-52cb0d20-d85a-4949-9bca-4ed0cac1aa08.png)
![image](https://user-images.githubusercontent.com/66630801/167531879-90106247-0092-4a4e-9eb9-9e387d5a4f87.png)

**Comment by the-run-philosophy** [comment]
有没有这样的基金会我不知道，但本repo暂时没有得到基金会的援助
希望基金会早日援助我们把网站建好充实内容lol

**Comment by AntEffect** [comment]
问:Ned干了什么？
答:他们支持了润学
问:润学又干了什么？
答:把tg干过的事情搞了个合订本

**Reply by lxn7022** [reply]
NED干了这个：
https://www.toutiao.com/article/7102803353993789956
